src/main/utility/pod.rs

```diff
@@ -278,3 +278,7 @@ unsafe impl Pod for libc::clone_args {}
 
 // shadow re-exports this definition from /usr/include/linux/tcp.h
 unsafe impl Pod for crate::cshadow::tcp_info {}
+
+// TODO: Move this module out of `main` so that `shadow_shmem` can use it,
+// and put this with the struct definition.
+unsafe impl Pod for shadow_shmem::allocator::ShMemBlockSerialized {}
```

Comment:
Maybe add a comment to the original struct definition at `c::ShMemBlockSerialized` stating that it must be POD.

Reply:
done